pkgs/tools/networking/yrd/default.nix

```diff
@@ -0,0 +1,28 @@
+{ stdenv, fetchFromGitHub, pythonPackages }:
+
+let
+  pname = "yrd";
+  version = "0.5.3";
+  sha256 = "1yx1hr8z4cvlb3yi24dwafs0nxq41k4q477jc9q24w61a0g662ps";
+
+in pythonPackages.buildPythonApplication {
+  name = "${pname}-${version}";
+
+  src = fetchFromGitHub {
+    owner = "kpcyrd";
+    repo = "${pname}";
+    rev = "v${version}";
+    inherit sha256;
+  };
+
+  propagatedBuildInputs = with pythonPackages; [ argh ];
+
+  meta = with stdenv.lib; {
+    description = "Cjdns swiss army knife";
+    maintainers = with maintainers; [ akru ];
+    platforms = platforms.linux;
+
+    homepage = https://github.com/kpcyrd/yrd;
+    inherit version;
```

Comment:
I don't think this is common attribute of `meta`. Could you check and remove it?